source/javascripts/core/models/BitriseYmlService.ts

```diff
@@ -232,6 +278,17 @@ function omitEmptyIfKeyNotExistsIn<T>(o: Record<string, T>, keys: string[]) {
 
 // PRIVATE FUNCTIONS
 
+function renameWorkflowInChains(workflowId: string, newWorkflowId: string, workflows: Workflows): Workflows {
+  return mapValues(workflows, (workflow) => {
```

Comment:
Do we need this `mapValues`?

Reply:
Yes. Because it traverse a Workflows (`{.wf1: {}, wf2: {} }`) type and returns an updated Workflows type as well...